cranelift/codegen/src/dominator_tree.rs

```diff
@@ -32,7 +423,7 @@ struct DomNode {
 }
 
 /// The dominator tree for a single function.
-pub struct DominatorTree {
+pub struct SimpleDominatorTree {
```

Comment:
This should be moved to a separate file to make the code cleaner. At the moment it looks messy with 2 separate algorithms in the same file.

Reply:
Moved to a separate file